app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  validates :body, presence: true
+  validates :author_id, presence: true
+  validates :post_id, presence: true
```

Comment:
- `belongs_to` association already adds presence validation. For Rails 5 there is a config named `Rails.application.config.active_record.belongs_to_required_by_default` is set always to `true`, so you don't have to add presence validation for both `author_id` and `post_id`. 
[1](https://til.hashrocket.com/posts/ca64beda59-rails-5-changed-belongsto-default-validation).

Reply:
great, good to know thanks!